File: Volume 26 (2600 - 2699)/2685/2685.c
#include <stdio.h>
 
int main() {
    double m;
    while (scanf("%lf", &m) != EOF) {
        if (m >= 360) m /= 360;
        if (m >= 0 && m < 90) printf("Bom Dia!!\n");
        else if (m >= 90 && m < 180) printf("Boa Tarde!!\n");
        else if (m >= 180 && m < 270) printf("Boa Noite!!\n");
        else if (m >= 270 && m < 360) printf("De Madrugada!!\n");
    }
    return 0;
}

File: Volume 26 (2600 - 2699)/2685/2685.rb
#!/usr/bin/ruby

while line=gets
    line = line.to_i
    if line >= 360
        line /= 360
    end
    if line >= 0 and line < 90
        puts "Bom Dia!!"
    elsif line >= 90 and line < 180
        puts "Boa Tarde!!"
    elsif line >= 180 and line < 270
        puts "Boa Noite!!"
    elsif line >= 270 and line < 360
        puts "De Madrugada!!"
    end
end